ocaml/idl/datamodel.ml

```diff
@@ -1216,6 +1216,8 @@ let _ =
     ~doc:"The update precheck stage failed: the server is of an incorrect version." ();
   error Api_errors.update_precheck_failed_out_of_space ["update"; "available_space"; "required_space "]
     ~doc:"The update precheck stage failed: the server does not have enough space." ();
+  error Api_errors.update_precheck_failed_gpgkey_not_imported ["update"]
+    ~doc:"The update precheck stage failed: the update required gpg key is not imported in the host." ();
```

Comment:
We use a GPG signature for validating the hotfix as a whole, and then the `yum` command checks GPG signatures on individual RPM packages.
This error corresponds to an error message from `yum`.
Therefore please would you make a change?
`the update required gpg key is not imported in the host.`
would be better as something like
`RPM package validation requires a GPG key that has not been imported on the host."